Code/GraphMol/Wrap/rough_test.py

```diff
@@ -3500,7 +3500,77 @@ def testGetSDText(self) :
         for pn in m.GetPropNames():
             self.assertTrue(nm.HasProp(pn))
             self.assertEqual(m.GetProp(pn),nm.GetProp(pn))
+            
+  
+  def testUnfoldedRDKFingerprint(self):
+    from rdkit.Chem import AllChem
+    
+    m = Chem.MolFromSmiles('c1ccccc1N')
+    fp = AllChem.UnfoldedRDKFingerprintCountBased(m)
+    fpDict = fp.GetNonzeroElements()
+    self.assertEquals(len(fpDict.items()),19)
+    self.assertTrue(fpDict.has_key(374073638))
```

Comment:
`has_key` is deprecated in python 3.x.
The right way to check this (works in python 2.7 too) is: 
`self.assertTrue(374073638 in fpDict)`